pom.xml

```diff
@@ -69,6 +69,12 @@
       <classifier>test-binder</classifier>
       <type>test-jar</type>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/net.sf.jsefa/jsefa -->
+    <dependency>
+      <groupId>net.sf.jsefa</groupId>
```

Comment:
@TommasoLencioni This library is too old. It hasn't been updated since 2014
https://mvnrepository.com/artifact/net.sf.jsefa/jsefa